Resources/Prototypes/DeltaV/Recipes/Lathes/security.yml

```diff
@@ -223,4 +223,3 @@
   materials:
     Steel: 1000
     Plastic: 800
-    
```

Comment:
why

Reply:
On master there is a random line 227, which I deleted. I believe that's why it looks wonky. do I need to untouch this?

Reply:
yes why would you delete it